Podcast: Episode 34 (#181)
nextflow-io/website: src/content/podcast/2024/ep34_jakob_zeitler_materials_science.md

episode: 34
description: We speak to Jakob from Matterhorn Studio about how they use Nextflow and Seqera Platform in materials science research.
date: 2024-04-02
type: podcast
subtype: Interview
youtubeid: -GzuKTMXUb0
image: /img/podcast_ep34.png
tags: nextflow,opensource
author: Developer advocates
icon: logo_podcast_channels.jpg
---

In this episode of Channels, we talk to [Jakob Zeitler](https://jakobzeitler.github.io/) - Head of R&D at [Matterhorn Studios](https://matterhorn.studio/).

We dive into how they use Nextflow and Seqera Platform for materials science research, paving the way for cheaper and more eco-friendly products for the future ⚗️ 👩🏻‍🔬 🔬 🌎

<!-- end-archive-description -->

Jakob is currently finishing his PhD at UCL, and he started Matterhorn STUDIO in 2022
to bring machine learning to materials science.

> You might remember Jakob's name from an [excellent talk](https://summit.nextflow.io/2023/barcelona/agenda/summit/oct-19-machine-learning-for-material-science/)
> that he gave at the 2022 Nextflow Summit in Barcelona.
> You can watch that video on [YouTube](https://youtu.be/kFDNRWXa_Mg?si=mZ-HoDnVrZ0hoqa3).

> We talk about how advances in materials science can bring many benefits to mankind,
> for example better materials to help the fight against climate change.
> However, the current paradigms are too slow, with and average 20 years form inception to market.

Author: ```suggestion
Join to hear about how advances in materials science can bring many benefits to mankind,
for example better materials to help the fight against climate change.
Jakob tells us about how the current paradigms are too slow, with and average 20 years
from inception to market.
```

---
title: "Nextflow in materials science: Jakob Zeitler"
episode: 34
description: We speak to Jakob from Matterhorn Studio about how they use Nextflow and Seqera Platform in materials science research.
date: 2024-04-02
type: podcast
subtype: Interview
youtubeid: -GzuKTMXUb0
image: /img/podcast_ep34.png
tags: nextflow,opensource
author: Developer advocates
icon: logo_podcast_channels.jpg
---

In this episode of Channels, we talk to [Jakob Zeitler](https://jakobzeitler.github.io/) - Head of R&D at [Matterhorn Studios](https://matterhorn.studio/).

We dive into how they use Nextflow and Seqera Platform for materials science research, paving the way for cheaper and more eco-friendly products for the future ⚗️ 👩🏻‍🔬 🔬 🌎

<!-- end-archive-description -->

> It may not be bioinformatics, but there's a lot that's in common!
> 

Author: ```suggestion
```

Jakob is currently finishing his PhD at UCL, and he started Matterhorn STUDIO in 2022

> to bring machine learning to materials science.
> 

Author: ```suggestion
to bring machine learning to materials science.

> You might remember Jakob's name from an [excellent talk](https://summit.nextflow.io/2023/barcelona/agenda/summit/oct-19-machine-learning-for-material-science/)
> that he gave at the 2022 Nextflow Summit in Barcelona.
> You can watch that video on [YouTube](https://youtu.be/kFDNRWXa_Mg?si=mZ-HoDnVrZ0hoqa3).

```

We talk about how advances in materials science can bring many benefits to mankind,
for example better materials to help the fight against climate change.
However, the current paradigms are too slow, with and average 20 years form inception to market.

Matterhorn STUDIO was founded to bring efficient experimentation to materials science,
using Nextflow to structure ML routines - specifically Bayesian Optimisation.

We talk about why Matterhorn STUDIO chose Nextflow to run their computational workloads,
and how they built on top of the Seqera Platform API to create their tools and abstract
the complexity of cloud computing away from users. All the time whilst staying true to
open-source science, with reproducible and portable analysis pipelines.

> You might remember Jakob's name from an [excellent talk](https://summit.nextflow.io/2023/barcelona/agenda/summit/oct-19-machine-learning-for-material-science/)
> that he gave at the 2022 Nextflow Summit in Barcelona.
> You can watch that video on [YouTube](https://youtu.be/kFDNRWXa_Mg?si=mZ-HoDnVrZ0hoqa3).

Author: ```suggestion
```
